$NetBSD: patch-af,v 1.3 2011/10/16 15:19:09 spz Exp $

xymonclient starts from rc.d, don't start it via xymon also

--- xymond/etcfiles/tasks.cfg.DIST.orig	2011-03-08 17:20:28.000000000 +0000
+++ xymond/etcfiles/tasks.cfg.DIST
@@ -180,6 +180,7 @@
 # Xymon server ?), but there's no need to have two xymonlaunch instances
 # running at the same time. So we'll just run it from here.
 [xymonclient]
+	DISABLED
 	ENVFILE @XYMONTOPDIR@/client/etc/xymonclient.cfg
 	NEEDS xymond
 	CMD @XYMONTOPDIR@/client/bin/xymonclient.sh
